### PR TITLE
3.8.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.7.1 (2019-10-10)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: This would be a minor version, yes?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: I was on the fence. I saw it as an update to existing functionality. Now that I go back and read the definition of a minor update, I see there's a strong argument for minor here. Changed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# 3.7.1 (2019-10-10)
+# 3.8.0 (2019-10-10)
 
 - Updated `getFocusableChildren()` method to make filtering optional for tab index and visibility. [#104](https://github.com/blackbaud/skyux-core/pull/104)
```
[the remark this answers is not in the document]
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# 3.8.0 (2019-10-10)
+
+- Updated the `getFocusableChildren()` method to make filtering optional for tab index and visibility. [#104](https://github.com/blackbaud/skyux-core/pull/104)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: Updated the method...on what?

**Pull request merged**